### PR TITLE
[343] Horizontal line colours

### PR DESCRIPTION
**Pull request by AbigailMcP**
## Changes in this PR

- Remove the section breaks from above main text on all pages (topic pages and resources)
- Add section breaks above feedback sections

## Guidance to review

https://trello.com/c/2GV9ReTV/343-remove-horizontal-break-lines-on-main-text-and-add-horizontal-break-line-above-share-your-ideas

- Check that the section break colours match which section they're in
- Check we're not missing any
- Check it looks ok on mobile

Screenshot 2024-03-06 at 16 31 34

### CODE REVIEW

**Comment by AbigailMcP** [comment]
[Remove horizontal break lines on main text and add horizontal break line above share your ideas](https://trello.com/c/2GV9ReTV/343-remove-horizontal-break-lines-on-main-text-and-add-horizontal-break-line-above-share-your-ideas)

File: content/assets/css/application.scss
Comment (mark-pkgflow) [inline content/assets/css/application.scss:1]: Nice!

**Review by mark-pkgflow — APPROVED** [review]
Nice Job

**Comment by claire-hughez** [comment]
beautiful!

**Comment by lauren-oseni-2** [comment]
Looks fab!

**Pull request merged**